src/test/java/org/takes/rs/RsWithCookieTest.java

```diff
@@ -38,22 +38,27 @@
 public final class RsWithCookieTest {
 
     /**
-     * RsWithCookie can add cookies.
+     * RsWithCookie can add cookies (even with several decorations).
```

Comment:
@Bertram25 Let's keep this unit test as it is. Instead, add a second test "addsMultipleCookies" with the behavior we're specifically trying to implement.